artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
```

Comment:
It shouldn't be hard-coded in, but it's probably better to pass it in from the command line to `main`. That way it doesn't matter what the file is called, and you could draw data from any file you want.